pkg/security/admission/admission.go

```diff
@@ -73,6 +74,13 @@ func (c *constraint) Admit(a admission.Attributes) error {
 		return nil
 	}
 
+	// if this is an update, see if we are only updating the ownerRef.  Garbage collection does this
+	// and we should allow it in general, since you had the power to update and the power to delete.
+	// The worst that happens is that you delete something, but you aren't controlling the privileged object itself
+	if a.GetOldObject() != nil && oadmission.IsOnlyMutatingOwnerRefs(a.GetObject(), a.GetOldObject()) {
```

Comment:
I meant if ownerRefs changes could be whitelisted for all objects without injecting it into specific admission plugins. First thing that comes to my mind is that we will add/modify  admission an restrict it for some objects without knowing about it. We don't have any tests to make sure ownerRefs are editable.